CordovaLib/cordova.js

```diff
@@ -1,5 +1,5 @@
 // Platform: ios
-// 538a985db128858c0a0eb4dd40fb9c8e5433fc94
+// cordova-js rel/6.0.0-10-g07379820
```

Comment:
This should be reverted.
```suggestion
// 538a985db128858c0a0eb4dd40fb9c8e5433fc94
```